blogposts/announcing-sourcegraph-3.1.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Announcing Sourcegraph 3.1'
+author: 'Ryan Blunden'
+publishDate: 2019-02-20T05:59-06:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-3.1
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: false
+---
+
+Sourcegraph 3.1 is now out! This release helps you create more powerful queries, improves code intelligence, adds new Sourcegraph extension APIs, and adds a huge amount of new documentation. We're also changing to scheduled releases on the 20th of each month.
+
+<!-- (ryan): Hyperlink to headings in above paragraph -->
+
+What is Sourcegraph? Sourcegraph is an open-source, self-hosted, cross-repository code search and navigation tool, with an efficient web interface and "feels-like-native" integration into your code host. [Install or upgrade Sourcegraph.](#install-or-upgrade)
+
+## Find more answers in code with the new query builder
+
+<!-- TODO (ryan): Below paragraph needs work as it sounds awkward -->
+
+We want to make it easier for developers to get the most of our code searching features. Did you know that in addition to code, you can search commit diffs, commit messages and symbols?
+
+Even experienced Sourcegraph users will find something new. Give it a try and let us know what you think via [GitHub](https://github.com/sourcegraph/sourcegraph/issues/new/choose) or [Twitter](https://twitter.com/srcgraph).
+
+<!-- TODO (ryan): GIF using the query builder -->
+
+## Code intelligence updates
+
+Code intelligence for [every language](https://sourcegraph.com/extensions?query=category%3A%22Programming+languages%22) continues to improve, the highlights being:
+
+- No longer showing "Go to definition" on comments, except for tokens that look like code (e.g. when surrounded by backticks, or when it looks like a function call `foo()`)
+- More accurate docstring extraction from single-line or multi-line block comments.
+- Indented sections of docstrings are rendered as code blocks.
+
+Improving code intelligence is a priority for us in  2019 so expect regular improvement updates in future releases. Want to improve support for your favorite language(s)? [Create an issue](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=&labels=&template=feature_request.md&title=Language%20extension%20improvement%20for).
+
+## Site admin improvements
+
+Administering Sourcegraph in an enterprise SSO environment is now easier thanks to a couple of small, but important features:
+
+- [Querying users by email address](https://sourcegraph.com/api/console#%7B%22query%22%3A%22%7B%5Cn%20%20user(email%3A%20%5C%22ryan%40sourcegraph.com%5C%22)%20%7B%5Cn%20%20%20%20id%5Cn%20%20%7D%5Cn%7D%5Cn%22%7D).
```

Comment:
I don't think this link is particularly helpful. Seems like this will more often be used on the **site-admin>Users** page lookup, rather than through the API anyway.